projects/marketing-site/components/Team/CoreTeam.tsx

```diff
@@ -82,27 +86,33 @@ export default function CoreTeam() {
                     </div>
 
                     <ul role="list" className="flex justify-center space-x-5">
-                      <li>
-                        <a
-                          href={person.twitterUrl}
-                          className="text-blue-500 hover:text-gray-300"
-                        >
-                          <span className="sr-only">Twitter</span>
-                          <svg
-                            className="h-5 w-5"
-                            aria-hidden="true"
-                            fill="currentColor"
-                            viewBox="0 0 20 20"
+                      {person.twitterUrl && (
+                        <li>
+                          <Link
+                            href={person.twitterUrl}
+                            className="text-blue-500 hover:text-gray-300"
```

Comment:
In light mode, if i hover to twitter, github, linkedin icon, it will disappear because it's the same color as the background
![Filmage 2023-01-14_233543](https://user-images.githubusercontent.com/13186215/212483805-00184719-6801-4c55-90c2-727c023406cd.gif)
Can you set className attribute to this 
```
className="text-blue-500 dark:hover:text-gray-300 hover:text-white"
```
Also apply with linkedin & github

Reply:
thanks! great catch!